src/software/firmware/srcs/pression.cpp

```diff
@@ -71,19 +91,12 @@ int16_t readPressureSensor(uint16_t centiSec) {
 #else
 
 int16_t readPressureSensor(uint16_t centiSec) {
-    int32_t rawPressure = static_cast<int32_t>(analogRead(PIN_PRESSURE_SENSOR));
-    int32_t delta = rawPressure - filteredRawPressure;
-
-    // Adjust delta so that the division result will be rounded away from zero.
-    // This is needed to guaranty that filteredRawPressure will reach
-    // rawPressure when it is constant.
-    delta += (delta > 0) ? (RAW_PRESSURE_FILTER_DIVIDER - 1) :
-                          -(RAW_PRESSURE_FILTER_DIVIDER - 1);
-    filteredRawPressure += delta / RAW_PRESSURE_FILTER_DIVIDER;
-
-    int16_t scaledRawPressure = filteredRawPressure * RAW_PRESSURE_TO_MMH20_NUM
-            / RAW_PRESSURE_TO_MMH20_DEN;
-    return max(scaledRawPressure - RAW_PRESSURE_TO_MMH20_CONSTANT, 0);
+    (void) centiSec;
+#ifndef UNIT_TEST
+    return convertSensor2Pressure(analogRead(PIN_PRESSURE_SENSOR));
+#else
+    return 0;
```

Comment:
Why do we want to return `0` when in unit tests? Is it because we can't have a call to `analogRead()` in the tests without mocking it?

Reply:
exactly